img/r/ringcentral.com.svg

```diff
@@ -0,0 +1,15 @@
+<path clip-rule="evenodd" d="M327.875 74.9904V87.7904H326.808C321.761 78.8571 311.265 72.6476 296.332 72.6476C274.465 72.6476 254.751 89.9428 254.751 120.457C254.751 150.971 274.18 168.457 296.332 168.457C310.713 168.457 319.189 162.609 325.684 152.705H326.713V172.362C326.713 182.857 320.884 187.714 311.361 187.714H265.646V207.79H310.922C335.989 207.79 349.399 193.79 349.399 170.857V74.9904H327.875ZM302.522 148.057C287.551 148.057 278.027 138.743 278.027 120.476C278.027 102.209 287.456 92.8762 302.522 92.8762C317.094 92.8762 326.808 102.19 326.808 120.476C326.808 138.762 317.189 148.057 302.522 148.057Z" fill="#0684BC" fill-rule="evenodd"></path>
+<path clip-rule="evenodd" d="M176.865 168.267V114.248C176.865 101.029 185.037 92.381 198.256 92.381C210.884 92.381 218.465 99.6762 218.465 113.962V168.267H240.998V110.362C240.998 86.8381 229.57 72.6667 206.218 72.6667C192.027 72.6667 181.056 79.0667 176.198 87.8095H175.418V74.9905H154.332V168.324L176.865 168.267Z" fill="#0684BC" fill-rule="evenodd"></path>
+<path d="M135.284 74.9905H112.751V168.267H135.284V74.9905Z" fill="#0684BC"></path>
+<path clip-rule="evenodd" d="M73.4938 103.943V103.029C87.2842 99.2191 96.6176 89.962 96.6176 69.162C96.6176 47.7905 80.6747 32.2286 51.1319 32.2286H0.998535V168.267H24.3128V114.248H52.6938C65.9128 114.248 72.5223 120.857 72.5223 134.076V168.362H95.8366V134.076C95.8366 114.838 87.6652 107.067 73.4938 103.943ZM49.8747 93.2572H24.3128V53.2572H49.8747C62.5033 53.2572 72.2176 58.5144 72.2176 73.2763C72.2176 88.0382 62.4081 93.2572 49.8747 93.2572Z" fill="#0684BC" fill-rule="evenodd"></path>
+<path clip-rule="evenodd" d="M124.008 60.0191C126.971 60.0229 129.868 59.1475 132.333 57.5039C134.798 55.8603 136.72 53.5222 137.855 50.7858C138.991 48.0493 139.289 45.0375 138.712 42.1316C138.135 39.2256 136.708 36.5562 134.614 34.4613C132.519 32.3663 129.849 30.9401 126.943 30.363C124.037 29.7859 121.025 30.084 118.289 31.2195C115.553 32.355 113.215 34.2769 111.571 36.7419C109.927 39.2068 109.052 42.104 109.056 45.0667C109.03 48.0324 109.895 50.9375 111.539 53.4062C113.182 55.8749 115.529 57.7937 118.275 58.9143C120.091 59.6692 122.042 60.0451 124.008 60.0191Z" fill="#FF7A00" fill-rule="evenodd"></path>
+<path clip-rule="evenodd" d="M106.827 3.58098C101.369 5.86562 96.4121 9.19834 92.2366 13.3905L102.808 23.9619C105.579 21.175 108.872 18.9612 112.499 17.4471C116.126 15.9331 120.016 15.1483 123.946 15.1377C127.876 15.1271 131.77 15.8908 135.405 17.3853C139.04 18.8798 142.346 21.0757 145.132 23.8476L155.703 13.2762C149.426 7.01155 141.433 2.74724 132.734 1.02169C124.035 -0.703868 115.02 0.186712 106.827 3.58098Z" fill="#FF7A00" fill-rule="evenodd"></path>
+</svg>
```

Comment:
That logo is hardly readable at the size we're usually using. This is their favicon in SVG format.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="150" height="150"><path d="M31.42 149.9c-.94-.06-1.17-.1-.8-.16.5-.08.49-.09-.15-.11A33.14 33.14 0 0 1 .42 119.58c-.02-.65-.02-.66-.12-.15-.25 1.35-.3-6.2-.3-44.39l.3-44.39c.1.5.1.5.12-.14A33.08 33.08 0 0 1 30.47.46c.65-.02.65-.02.15-.12C29.26.1 36.82.04 75 .04l44.4.3c-.51.1-.51.1.14.12a33.14 33.14 0 0 1 30.05 30.05c.02.64.02.65.11.14.26-1.35.3 6.2.3 44.39l-.3 44.39c-.1-.5-.1-.5-.11.14a33.14 33.14 0 0 1-30.05 30.05c-.65.02-.65.03-.15.12.35.06-.1.13-1.39.2-2.24.12-84.65.09-86.58-.04z" fill="#fe7900"/><path d="M120.74 137.04c6.48-1.5 11.59-5.42 14.44-11.1a23.2 23.2 0 0 0 1.83-5.24l.33-1.5V30.88l-.33-1.49a20.97 20.97 0 0 0-16.35-16.34l-1.49-.33H30.84l-1.5.33A20.97 20.97 0 0 0 13 29.38l-.33 1.5v88.33l.33 1.5c1.99 8.96 8.97 15.42 17.9 16.58.53.07 20.64.11 44.69.1l43.73-.02z" fill="#fff"/><path d="M33.91 117.1c-.16-.08-.46-.3-.66-.52l-.36-.4-.04-35.33c-.05-38.25-.07-36.91.72-39.47 1.48-4.8 5.22-7.7 10.83-8.38 1.01-.12 9.44-.15 31.56-.12 33.48.06 30.7-.02 33.52 1a11.38 11.38 0 0 1 6.96 7.5c.75 2.46.77 2.88.71 20.55l-.3 17.4a20.69 20.69 0 0 1-1.72 4.45 20.59 20.59 0 0 1-4.4 4.42c-.46.26-3.15 1.53-6 2.81s-5.31 2.48-5.5 2.65c-.78.74-.88.54 6.04 11.72l6.46 10.7c.06.3-.02.5-.3.77l-.38.38h-8.99c-10.1 0-9.83.03-11-1.08-.45-.41-2.72-3.85-7.11-10.75L77.07 94.5a16.07 16.07 0 0 1-1.54-5.06c-.18-1.27-.23-3-.23-7.45V76.2l.43-.43.42-.43h9.65c10.65 0 10.24.04 10.95-.92.3-.41.3-.56.34-10.68.04-10.01.04-10.28-.25-10.87-.59-1.21 1.42-1.11-21.86-1.11l-21.17.28a2.2 2.2 0 0 0-.66.63c-.16.27-.2 6.2-.27 31.94l-.07 31.6-.48.48-.48.48-8.82.03c-4.85.01-8.96-.03-9.12-.1z" fill="#0684bc"/></svg>
```